nomenclature/cli.py

```diff
@@ -27,6 +29,11 @@ def cli_valid_project(path: Path):
     3. Test that all model mappings in `mappings/` can be correctly parsed as a
        :class:`RegionProcessor` object. This includes a check that all regions mentioned
        in a model mapping are defined in the region codelist.
+
+    Example
+    -------
+    $ nomenclature validate-project . --mappings map --definitions def
```

Comment:
Maybe change to reflect that these are folder names and turn order to follow docs structure...
```suggestion
    $ nomenclature validate-project .  --definitions <def-folder> --mappings <mapping-folder>
```